generate_workspace/src/test/java/com/google/devtools/build/workspace/maven/ResolverTest.java

```diff
@@ -145,6 +145,26 @@ public void dependencyManagementWins() throws Exception {
   }
 
   @Test
```

Comment:
Can you include a test when the depmgt includes the version range

```java
// basic example
Dependency managed = getDependency("a:b:[1.0, 4.0)");
Dependency version = getDependency("a:b:2.0");

// gnarlier example
Dependency managed = getDependency("a:b:[1.0, 4.0)");
Dependency version = getDependency("a:b:[1.2, 3.0]");
```
Currently, I am unsure how that will behave.